parser/lexer/state.go

```diff
@@ -119,6 +124,8 @@ loop:
 				return not
 			case "in", "or", "and", "matches", "contains", "startsWith", "endsWith":
 				l.emit(Operator)
+			case "env":
+				return literalIdentifier
```

Comment:
Let’s implement this without altering lexer.

Reply:
I feel that handling tokens within lexer is a clear, simple way to accomplish this.  Is the concern that too much interpretation of "meaning" of expressions moves into lexer?  Other keywords in, matches, contrains, startWith, endsWith are also interpreted within lexer.  I will look at an alternative.  Perhaps env can be a normal indexed identifier that parser will autmatically point at "env".  I suspect that will cause copmlications worrying about the type of the underlying value that the index returns; we'll see.

Reply:
env is just a variable.